src/test/kotlin/it/pagopa/ecommerce/payment/requests/services/CartsServiceTests.kt

```diff
@@ -65,7 +65,7 @@ class CartsServiceTests {
 
   @Test
   fun `post cart ko with multiple payment notices`() = runTest {
-    val request = CartRequests.withMultiplePaymentNotice()
+    val request = CartRequests.withMultiplePaymentNotice(6)
```

Comment:
And here use the extracted constant +1

Reply:
Fixed with https://github.com/pagopa/pagopa-ecommerce-payment-requests-service/pull/52/commits/8335d6301c35a3474d916793cd9bff453e902939